### DISCUSSION TITLE
airflow.cfg Configuration does not take effect,What can I do

### DISCUSSION THREAD

**Discussion by tsword-come** [apache/airflow#21113] [Q&A]
<img width="699" alt="企业微信截图_16431714415471" src="https://user-images.githubusercontent.com/84377443/151104524-c725059c-3df1-4210-91e6-41e4723a19b2.png">
<img width="669" alt="企业微信截图_16431714936495" src="https://user-images.githubusercontent.com/84377443/151104536-b03249a9-333d-4f1a-a91a-f60c8e30e945.png">

**Comment by potiuk** [comment]
Make sure (and check every single thing in there - your file is in the right place and that it is acessible to your airflow https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html

Explore other options in the how to if you are lost.

**Reply by potiuk** [reply]
Check your home, env vars etc - but rigorousely follow what is described in the docs.

**Reply by tsword-come** [reply]
Thank you，I'll check it carefully
